src/Component/Carousel.php

```diff
@@ -0,0 +1,173 @@
+<?php
+/**
+ * Class Carousel.
+ *
+ * @package AMP
```

Comment:
Question: The root namespace is `Amp` but the package is `AMP`. Does this matter?

Also: Should the root namespace actually be `AMP`? Now that “AMP” is no longer an acronym of “Accelerated Mobile Pages” it's possible it may eventually be spelled “Amp” but this is not the case today. 

/cc @schlessera